webapp/src/http_handlers/sessions/sessions_tracker.rs

```diff
@@ -61,45 +87,108 @@ impl SessionsTracker
         }
     }
 
-    /// Builder method, that sets sessions life duration
+    /// ## Builder method, that sets sessions life duration.
+    /// ----
+    /// Args:
+    /// ---
+    /// * `duration` - new `life_duration` value.
     pub fn life_duration(mut self, duration: &Duration) -> Self
     {
         self.life_duration = duration.to_owned();
         self
     }
 
-    /// Returns a ref to a tracked session by its id
-    pub fn get_session(&self, id: &u128) -> Option<&Session>
+    /// ## Locks `sessions_data`.
+    /// ----
+    /// ## Returns:
+    /// `MutexGuard` for `sessions`. If the mutex is poisoned - it will end the process with the exit code of 1.
+    fn lock(&self) -> MutexGuard<HashMap<u128, Session>>
```

Comment:
is it possible to rename this function to something like `get_sessions()` or `mp_safe_sessions()` (mp stands for multi processor)?

Reply:
I think of renaming it to `lock_sessions`. I don't really want to get rid of the _lock_ part, since the function works with a mutex and returns a `MutexGuard`.